src/dmd/dsymbolsem.d

```diff
@@ -4317,9 +4317,10 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         // `@disable new();` should not be deprecated
         if (!nd.isDisabled() && !nd.isDeprecated())
         {
-            // @@@DEPRECATED_2.084@@@
-            // Should be changed to an error in 2.084
-            deprecation(nd.loc, "class allocators have been deprecated, consider moving the allocation strategy outside of the class");
+            // @@@DEPRECATED_2.091@@@
+            // Made an error in 2.087.
+            // Should be removed in 2.091
+            error(nd.loc, "class allocators are obsolete, consider moving the allocation strategy outside of the class");
```

Comment:
Same, should be outside the disabled / deprecated check

Reply:
We still need to allow for `@disable new();`.  I removed the check for `deprecated` though.